--- a/feeds/luci/luci.mk
+++ b/feeds/luci/luci.mk
@@ -84,7 +84,7 @@ define findrev
       set -- $$(git log -1 --format="%ct %h" --abbrev=7 -- $(if $(1),. ':(exclude)po',po)); \
       if [ -n "$$1" ]; then
         secs="$$(($$1 % 86400))"; \
-        yday="$$(date --utc --date="@$$1" "+%y.%j")"; \
+        yday="$$(date --utc --date="@$$(($$1 + 365*24*60*60))" "+%y.%j")"; \
         printf '%s.%05d~%s' "$$yday" "$$secs" "$$2"; \
       else \
         echo "0"; \
